pages/faq.tsx

```diff
@@ -91,6 +91,11 @@ const questions: ReadonlyArray<{
       </div>
     ),
   },
+  {
+    question: `I mined a block, but I'm not getting points for it. Why is that?`,
+    id: `mined-blocks-no-points`,
+    answer: `Even if your miner successfully mined a block, there's a chance that someone else has mined a competing heavier block that the network has chosen over your block. This is the difference of a block being on a fork vs on the main, or canonical, chain. Only blocks accepted as part of the canonical chain earn you points.`,
```

Comment:
Minor, but:
1. `vs.` with period to indicate abbreviation
2. `main (or canonical) chain` (not required but when I read this I read it haltingly / oddly)

Reply:
Elena is out and we can put in future changes to this. I'll merge it for now but feel free to put up a PR with any grammatical changes you would like.